drivers/video/isx019.c

```diff
@@ -797,6 +811,122 @@ static const int32_t g_isx019_wbmode[] =
 
 #define NR_WBMODE (sizeof(g_isx019_wbmode) / sizeof(int32_t))
 
+static const int32_t g_isx019_iso[] =
+{
+  1000,     /* ISO1 */
+  1200,     /* ISO1.2 */
+  1600,     /* ISO1.6 */
+  2000,     /* ISO2 */
+  2500,     /* ISO2.5 */
+  3000,     /* ISO3 */
+  4000,     /* ISO4 */
+  5000,     /* ISO5 */
+  6000,     /* ISO6 */
+  8000,     /* ISO8 */
+  10000,    /* ISO10 */
+  12000,    /* ISO12 */
+  16000,    /* ISO16 */
+  20000,    /* ISO20 */
+  25000,    /* ISO25 */
+  32000,    /* ISO32 */
+  40000,    /* ISO40 */
+  50000,    /* ISO50 */
+  64000,    /* ISO64 */
+  80000,    /* ISO80 */
+  100000,   /* ISO100 */
+  125000,   /* ISO125 */
+  160000,   /* ISO160 */
+  200000,   /* ISO200 */
+  250000,   /* ISO250 */
+  320000,   /* ISO320 */
+  400000,   /* ISO400 */
+  500000,   /* ISO500 */
+  640000,   /* ISO640 */
+  800000,   /* ISO800 */
+  1000000,  /* ISO1000 */
+  1250000,  /* ISO1250 */
+  1600000,  /* ISO1600 */
+  2000000,  /* ISO2000 */
+  2500000,  /* ISO2500 */
+  3200000,  /* ISO3200 */
+  4000000,  /* ISO4000 */
+  5000000,  /* ISO5000 */
+};
+
+#define NR_ISO (sizeof(g_isx019_iso) / sizeof(int32_t))
+
+/* Gain values corresponding to each elements of g_isx019_iso table.
+ * This needs to have the same size of g_isx019_iso.
```

Comment:
```suggestion
 * This needs to have the same size as g_isx019_iso.
```
(Also very minor nit!)

Reply:
@hartmannathan Fixed it.